org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.raml

```diff
@@ -188,3 +193,24 @@ traits:
         body:
           application/json:
           example: {}
+
+/?$availability: |
```

Comment:
I think /?$availability can be centrally defined much like /subscription

resourceTypes:
  - !include org.opent2t.availability.yaml

...

/?$availability: 
  type: {availabilityType}

and then have a file opent2t.base-schemas/org.opent2t.availability.yaml 
containing the details

availabilityType:
  displayName: Availability 
  get:
    responses:
      200:
        body:
          application/json:
            schema: !include opent2t.platform-schema.json
            example:  |
              {
                "opent2t": {
                  "schema": "org.opent2t.sample.something.superpopular",
                  "translator": "opent2t-translator-com-opent2t-p-something@0.1.1",
                  "controlId": 123
                },
                "availability": "online",
                "pi": "2AFFBDC2-12B5-49CB-BE40-2C0D8CE4CC8C",
                "rt": ["org.opent2t.sample.something.superpopular"],
                "entities": []
              }